opentracing/config_test.go

```diff
@@ -0,0 +1,67 @@
+package opentracing
+
+import (
+	"testing"
+
+	ot "github.com/opentracing/opentracing-go"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConfigurationDefaults(t *testing.T) {
+	assert := assert.New(t)
+
+	config := NewConfiguration()
+	assert.Equal(true, config.Enabled)
+	assert.Equal(false, config.Debug)
+	assert.Equal("opentracing.test", config.ServiceName)
+	assert.Equal("localhost", config.AgentHostname)
+	assert.Equal("8126", config.AgentPort)
+}
+
+func TestConfiguration(t *testing.T) {
+	assert := assert.New(t)
+
+	config := NewConfiguration()
+	config.SampleRate = 0
+	config.AgentHostname = "ddagent.consul.local"
+	config.AgentPort = "58126"
+	tracer, closer, err := NewTracer(config)
+	assert.NotNil(tracer)
+	assert.NotNil(closer)
+	assert.Nil(err)
+}
+
+func TestTracerConstructor(t *testing.T) {
```

Comment:
This basically does all that `TestConfiguration` seems to do, let's merge the two?